_events/2022/2022-08-dei-speaker-series.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Diversity in Open-source Software
```

Comment:
```suggestion
title: Diversity in Open-Source Software
```
just a suggestion

Reply:
Actually saw that and then forgot to correct. Thank you for catching this.